src/encoded/static/components/viz/BarPlot/UIControlsWrapper.js

```diff
@@ -29,8 +30,8 @@ export class UIControlsWrapper extends React.PureComponent {
 
     static contextualView = memoize(function(href){
         if (href){
-            // Hide on homepage.
-            const hrefParts = url.parse(href);
+            // Hide on homepage. I guess could have used PageTitle.isHomePage here instead.
```

Comment:
Out of curiosity, would there be a difference in functionality between the two?

Reply:
Nah... just realizing there's a `PageTitle.isHomePage` func that literally does the same thing. Self note to cleanup some code :-P.

Actually on CGAP we've refactored the PageTitleSection to be much cleaner / customizable that would wanna propagate to 4DN repo and this would likely entail getting rid of `PageTitle.isHomePage` so current approach (if hrefParts.pathname === "/") makes more sense prly.